packages/lint/package.json

```diff
@@ -28,13 +28,13 @@
     "@commitlint/parse": "9.1.1",
     "@hapi/joi": "17.1.1",
     "js-yaml": "3.14.0",
-    "tslib": "2.0.0"
-  },
-  "devDependencies": {
+    "tslib": "2.0.0",
     "@commitlint/types": "9.1.1",
     "@types/hapi__joi": "17.1.3",
+    "@types/js-yaml": "3.12.5"
```

Comment:
Are we exposing these types in our public APIs?

Reply:
yes